test/scalecube-transport/provider/PostMessageProvider.test.js

```diff
@@ -0,0 +1,37 @@
+import { Transport } from '../../../src/scalecube-transport/Transport';
+import { PostMessageProvider } from "../../../src/scalecube-transport/provider/PostMessageProvider";
+import { errors } from '../../../src/scalecube-transport/errors';
+import { setWorkers, removeWorkers, httpURI as URI } from '../utils';
+
+describe('Tests specifically for PostMessage provider', () => {
+  setWorkers(URI);
+
+  const testInvalidWorker = (worker) => {
+    const URIWithInvalidWorker = 'https://localhost:4000';
+    window.workers[URIWithInvalidWorker] = worker;
+    const transport = new Transport();
+    return expect(transport.setProvider(PostMessageProvider, { URI: URIWithInvalidWorker }))
+      .rejects.toEqual(new Error(errors.connectionRefused));
+  };
+
+  afterAll(() => {
+    removeWorkers();
+  });
+
+  it ('If an item for a provided URI is string instead of a Worker, an error is emitted', () => {
```

Comment:
it.each([777, 'Invalid worker',() => 555,  test: 'test' }]) will be better
https://jestjs.io/docs/en/api#describeeachtable-name-fn